src/julia.h

```diff
@@ -1431,6 +1431,8 @@ typedef struct _jl_tls_states_t {
     jl_task_t *root_task;
     jl_value_t *volatile task_arg_in_transit;
     void *stackbase;
+    char *stack_lo;
+    char *stack_hi;
```

Comment:
IIUC `char*` is the standard type in C for pointer arithmetic and is allowed to alias anything.